_includes/js/geopoints.md

```diff
@@ -40,7 +40,40 @@ query.find({
 
  At this point `placesObjects` will be an array of objects ordered by distance (nearest to farthest) from `userGeoPoint`. Note that if an additional `ascending()`/`descending()` order-by constraint is applied, it will take precedence over the distance ordering.
 
-To limit the results using distance, check out `withinMiles`, `withinKilometers`, and `withinRadians`.
+To limit the results using distance, check out `withinMiles`, `withinKilometers`, and `withinRadians`. Use the `sorted` parameter to sort the results by distance ascending.
```

Comment:
> Use the `sorted` parameter to sort the results by distance ascending.

suggestion: By default, the results will come back sorted by their distance and you can disable it with the sorted parameter.